src/services/FileInfo.ts

```diff
@@ -64,9 +67,12 @@ export default async function(path: string, options = {}): Promise<FileInfo> {
 
 /**
  * Retrieve the files list
+ * @param origin
+ * @param path
+ * @param options
  */
 export async function rawStat(origin: string, path: string, options = {}) {
-	const response = await createClient(origin).stat(path, {
+	const response = await createClient(origin, { headers: { requesttoken: getRequestToken() || '' } }).stat(path, {
```

Comment:
https://github.com/nextcloud/viewer/blob/2016523aa5f70e61f678cb8b8264a4be47669ee5/src/services/WebdavClient.ts#L30

Reply:
What about this?
We already inject the token in the client, do you actually needs to set it again?

Reply:
This client only work with `/dav/files` endpoints, and I need it to work with other endpoints like `/dav/files_versions`.
Or am I missing something ?

Reply:
Sorry, I saw the `import { getClient } from './WebdavClient'` and assumed you were using it as well, didn't realize the createClient.

The rawStat `createClient` is confusing. No other dav implementation in Viewer accept out-of-user-root paths
I didn't see that in the [PR that added this](https://github.com/nextcloud/viewer/pull/1709), I would have blocked it otherwise.

Viewer does NOT support files outside of the user root anyway ? Any reason you're not reusing our `WebdavClient.ts` then?

Reply:
Viewer need to be able to edit pictures from `/dav/photos/user/albums` and equivalent.
It will also need to be able display files' versions.

Any reason viewer should not support files outside of users' root ?

Reply:
> Any reason viewer should not support files outside of users' root ?

Not implemented.
We need to implement the entire Viewer using the `File/Folder` new standard.
That way we'll know what to expect.

Let's leave it like that, it's a bit hacky but t works™ :)

Reply:
> Any reason viewer should not support files outside of users' root ?

Inconsistency in what the backend supports (dav properties)

Reply:
> It will also need to be able display files' versions.

Well, let's have a call about this :see_no_evil: